### ISSUE TITLE
Apply new colors for a total of eight (8) categories

### ISSUE THREAD

**Issue by kgracey** [parallaxinc/BlocklyProp#226]
Hello, 

We would like to apply some new colors in BlocklyProp. I realize there's been much discussion around colors so far - and even some more this afternoon between Matt and I. I've also spent the better part of a week looking at how Blockly colors are used in general and am trying to leverage some experience from others. 

Bottom line up front! These are our new colors - and there are eight of them! 

![palette](https://cloud.githubusercontent.com/assets/9534148/15792435/12366eb4-2992-11e6-8c7d-9643dca61876.png)

Would love to have a gut reaction from @DarkmatterVale @cbradyatinquire @PropGit but also ask that @michellampo apply them as soon as possible in the following manner:

HSV (155,75,60) Input / Output  
HSV (180,75,60) Sensors
HSV (200,75,60) Communicate
HSV (220,75,60) Programming (now includes the Functions and Variables categories) + Robot
HSV (240,75,60) Activity Board
HSV (260,75,60) Hackable Electronic Badge
HSV (280,75,60) Actuator / Output
HSV (300,75,60) Communicate

Thanks! Ken

**Comment by michel-cf** [comment]
A little preview of blocks of all categories:

![image](https://cloud.githubusercontent.com/assets/5745991/15793278/29ad4f94-29e3-11e6-8da9-a13923396894.png)

**Comment by kgracey** [comment]
Feeling better already. Please be prepared for more adjustments!

Ken Gracey  |  Parallax Inc.  |  Direct: (916) 625-3010 <tel:%28916%29%20624-8333>  |  www.parallax.com <http://www.parallax.com/>  |  @ParallaxKen <https://twitter.com/parallaxken>

**Comment by michel-cf** [comment]
![image](https://cloud.githubusercontent.com/assets/5745991/15793522/c7edcc3c-29e4-11e6-97ae-c373c0e365bd.png)

This should be according to plan. (I also made the program something that could work. Excluding some of the blocks of other catergories. As you can see, much logic makes it monotone)

**Comment by kgracey** [comment]
That looks good so far!

*Matthew Matz  |  STEM/Robotics Educator*
Parallax Inc.  |  Direct: 916-625-3019  |  www.parallax.com  |  @M
atzElectronics <http://twitter.com/MatzElectronics>

**Comment by MatzElectronics** [comment]
HSV (155,75,60) Input / Output
HSV (180,75,60) Sensors
HSV (200,75,60) Communicate
HSV (220,75,60) Programming (now includes the Functions and Variables categories) + Robot
HSV (240,75,60) Activity Board
HSV (260,75,60) Hackable Electronic Badge
~~HSV (280,75,60)~~ Actuator / Output
~~HSV (300,75,60)~~ Communicate
*HSV (275,75,60)*
*HSV (290,75,60)*
*HSV (305,75,60)*
*HSV (320,75,60)*

That gets us two more colors without making it pukey, but the last one (320) should be used on something that doesn't get used very often...

**Comment by MatzElectronics** [comment]
Gave it some serious thought this evening.  Here's what I propose:
`        'programming': 220,
        'math': 275,
        'binary': 275,
        'io': 200,
        'protocols': 320,
        'ab': 305,
        'input': 155,
        'output': 180,
        'robot': 290,
        'heb': 290,
        'functions': 240,
        'variables': 260,`

**Comment by kgracey** [comment]
Are you reorganizing colors AND categories with this suggestion?

Ken

**Comment by MatzElectronics** [comment]
Colors yes, categories no.  In your original suggestion, you doubled up
some things that were commonly used together, so a typical program would
likely end up looking very bland.

Another thing I realized is that colors should be doubled in an either or
case.  For example, you would be programming either a badge or a robot, not
both, so they should be the same color.

As for the suggestion itself, I went into the actual code to see exactly
how your original suggestion got translated...

Does that all make sense?
On Jun 5, 2016 7:33 AM, "Ken Gracey" <notifications@github.com> wrote:

Are you reorganizing colors AND categories with this suggestion?

Ken

Sent from my iPhone

> On Jun 4, 2016, at 10:52 PM, Matthew Matz <notifications@github.com>
wrote:
>
> Gave it some serious thought this evening. Here's what I propose:
> 'programming': 220,
> 'math': 275,
> 'binary': 275,
> 'io': 200,
> 'protocols': 320,
> 'ab': 305,
> 'input': 155,
> 'output': 180,
> 'robot': 290,
> 'heb': 290,
> 'functions': 240,
> 'variables': 260,

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/parallaxinc/BlocklyProp/issues/226#issuecomment-223816418>,
or mute the thread
<https://github.com/notifications/unsubscribe/AS0quOe17knsw2V9lRLJu9Ndxvm4y-jgks5qIt4xgaJpZM4It6ac>
.
